.travis.yml

```diff
@@ -83,5 +83,10 @@ matrix:
 # Use travis_retry during install to reduce the impact of network failures.
 before_install: travis_retry ./Support/Testing/Travis/before-install.py
 install:        travis_retry ./Support/Testing/Travis/install.py
+before_script:
```

Comment:
I think @fjricci is just doing some preliminary testing work right now. That will probably end up in a separate file like the other steps when it gets closer to being ready.